cloud-data-protection/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {Route} from 'react-router';
+import {Layout} from './components/layout';
+import {Home} from './components/home';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import { Home } from "./components/home";
```